Project.toml

```
@@ -1,7 +1,7 @@
 name = "MakieDraw"
 uuid = "8315f7d3-d5d2-4d16-bebe-3aa4ef51e4f8"
 authors = ["Rafael Schouten <rafaelschouten@gmail.com>"]
-version = "0.2.1"
+version = "0.2.2"
```

Should this be breaking?  It will cause a lot of workflows using eg CanvasSelect to fail...
```suggestion
version = "0.3.0"
```

---

Yeah I bumped the version before I broke everything